### DISCUSSION TITLE
ScriptReference/Screen

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#594] [Notes]
# ScriptReference/Screen

https://docs.unity3d.com/ScriptReference/Screen.html

**Comment by snlehton** [comment]
`Screen.currentResolution` is not the size of the viewport. If you need the size of the viewport, use `Screen.width` and `Screen.height`. If you need `Resolution` object, you can use this
`return new Resolution()  {  width = Screen.width, height = Screen.height  };`
